src/components/reservation/ReservationModalBody.tsx

```diff
@@ -126,7 +126,11 @@ export const ReservationModalBody = ({
   const { mutate: mutateAddReservations } = useAddReservationsV2();
   const { mutate: mutateOpenOrder } = useOpenOrderMutation();
   const userResponse = usePatronData();
-  const holdingsResponse = useGetHoldings({ faustIds, config });
+  const holdingsResponse = useGetHoldings({
+    faustIds,
+    config,
+    blacklist: "availability"
+  });
```

Comment:
I am unsure whether this is as requested.

This previously used the default value `false` for `useAvailabilityBlacklist`. This was mapped to the `pickup` type. Now it is the `availability` type.